src/site/stages/build/page-builder/schemas/page.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: {
+      $ref: 'GenericNestedString',
+    },
+    field_intro_text: {
+      $ref: 'GenericNestedString',
+    },
+    field_description: {
+      $ref: 'GenericNestedString',
+    },
+    field_featured_content: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_content_block: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_alert: {
+      type: 'array',
+    },
+    field_related_links: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_administration: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_page_last_built: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string', format: 'custom-date-time' },
+        },
+      },
+    },
+  },
+  required: [
+    'title',
+    'field_intro_text',
+    'field_description',
+    'field_featured_content',
+    'field_content_block',
+    'field_alert',
+    'field_related_links',
+    'field_administration',
+    'field_page_last_built',
+  ],
```

Comment:
If this is supposed to match what we are extracting in the filtering step I can see it quickly getting out of sync.  Should we have a single source of truth for our required/filtered items?

Reply:
Good call! Yes. I'll pull in the filters. :+1: